src/components/lender/active-loans/ActiveLoansTable.tsx

```
@@ -1,211 +1,727 @@
+              </Button>
+
+              {isDropdownOpen && (
+                <div className="absolute right-0 mt-1 w-[160px] bg-[#0f172a] border border-[#1e2d3d] rounded-md overflow-hidden z-50 animate-fadeIn shadow-lg dropdown-container">
+                  <div className="py-1 border-b border-[#1e2d3d]">
+                    <button
+                      onClick={() => {
+                        setStatusFilter("All Status");
+                        setIsDropdownOpen(false);
+                      }}
+                      className="w-full px-4 py-2 text-left text-[#0ff] font-medium hover:bg-[#1e2d3d] transition-colors"
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)